corehq/apps/geospatial/routing_solvers/pulp.py

```diff
@@ -1,13 +1,13 @@
 import haversine
 
 import requests
+import pulp
 
-from ortools.linear_solver import pywraplp
 from django.conf import settings
 from .mapbox_optimize import validate_routing_request
 
 
-class ORToolsRadialDistanceSolver:
+class RadialDistanceSolver:
```

Comment:
is this in use currently at all?

Reply:
No. A ticket I have in progress makes use of this.